presets/properties.py

```diff
@@ -46,7 +46,7 @@ def get_enum_items(self, context):
 
     @classmethod
     def get_from_path(cls, lib_path):
-        group_path,preset = lib_path.rsplit('/', 1)
+        group_path, preset = os.path.split(lib_path)
```

Comment:
I actually think this doesn't give you what you want right?  maybe you'd do

tokens = os.path.split(lib_path)
preset = tokens[-1]
group_path = os.path.join(tokens[:-1])